addons/web/static/src/legacy/utils.js

```diff
@@ -40,9 +45,11 @@ export function makeLegacyRpcService(legacyEnv) {
  */
 export function makeLegacyDialogMappingService(legacyEnv) {
     return {
-        dependencies: ["ui", "hotkey"],
+        dependencies: ["ui", "hotkey", "overlay"],
         start(env) {
-            const { ui, hotkey } = env.services;
+            const { ui, hotkey, overlay } = env.services;
+
+            overlay.add(LegacyDialogContainer, {});
```

Comment:
If I understand correctly, if a legacy dialog and a standard dialog are both open, the legacy one would always be behind.

Is that right ? Do we have cases where this is an issue ?